### ISSUE TITLE
blip.tv not supported

### ISSUE THREAD

**Issue by GoogleCodeExporter** [MiraculousLegendas/get-flash-videos#63]
```
The site http://blip.tv is not supported.

```

Original issue reported on code.google.com by `andi.g...@gmail.com` on 8 Feb 2010 at 11:00

**Comment by GoogleCodeExporter** [comment]
```
Looks like some of the URLs will work and some won't, there seems to be three 
different formats for links to videos within the site, one of them works and 
the 
others do not. 
For example this url works-
http://blip.tv/play/AbqKQwI

These do not-
http://blip.tv/file/1183244
http://blip.tv/file/3379568?
utm_source=episodepg_featured&utm_medium=episodepg_featured

Debug info:
Downloading http://blip.tv/file/1183244
-> GET http://blip.tv/file/1183244
<- 200 text/html; charset=utf-8 (35130)
Using method 'blip' for http://blip.tv/file/1183244
-> GET http://blip.tv/file/1183244
<- 200 text/html; charset=utf-8 (35130)
Error: No ID found

Downloading http://blip.tv/file/3379568?utm_source=episodepg_featured
-> GET http://blip.tv/file/3379568?utm_source=episodepg_featured
<- 200 text/html; charset=utf-8 (34825)
Using method 'blip' for 
http://blip.tv/file/3379568?utm_source=episodepg_featured
-> GET http://blip.tv/file/3379568?utm_source=episodepg_featured
<- 200 text/html; charset=utf-8 (34825)
Error: No ID found

```

Original comment by `tryagain...@gmail.com` on 23 Mar 2010 at 4:36

**Comment by GoogleCodeExporter** [comment]
```
Fixed in git 4c1620a.
```

Original comment by `zakflash...@gmail.com` on 25 May 2010 at 4:59
- Changed state: **Verified**
